davidsymonds/David_Symonds.md

```diff
@@ -0,0 +1,37 @@
+
+Wikipedia describes The Community Charge commonly known as the poll tax as “a system of taxation introduced by Margaret Thatcher's government in replacement of domestic rates in Scotland from 1989, prior to its introduction in England and Wales from 1990. It provided for a single flat-rate per-capita tax on every adult, at a rate set by the local authority. The charge was replaced by Council Tax in 1993, two years after its abolition was announced.”  Its introduction lead to riots, punk rock songs, the fall of the one of the UKs most successful Prime Ministers and my move into Software Testing.
+
+I had accepted a job with British Telecom (BT) as a test manager on a poll tax system for 14 local authorities. At that time there were only two telecom companies in the UK. The largest BT was contracted to provide a system to package and print the poll tax bills, payment books and ancillary documents that local authorities sent out each year. The system also printed the individual Local Authorities coats of arms on the documents. This project had started in the previous October. It was running late and the documents had to be produced before the beginning of April. I and my six colleagues had to get the project back on track. To this end we worked the extended hours expected nowadays from a Start-up.  
+
+One night I was in the office with a young colleague from the logo design team, correcting any of the flaws in the coats of arms that caused due to scanning imperfections. I looked up and noticed that his shoulders had slumped and he had his head in his hands. I went over to check everything was okay and he was as white as a sheet. He told me that while working on the coat of arms for the 14th local authority he had accidentally deleted those that had been successfully created for other 13. I asked him if there was a backup, to which he replied no. 
+
+1990 was sometime before the internet was useful for troubleshooting this type of problem, so as was my habit I referred to my personal engineering notes that I had kept since 1968. Here I found that a process existed within Norton Utilities for retrieving deleted files. As luck would have it, there was a copy of Norton on site. We retrieved all of the data in about three hours.
+
+In early March 1990 we moved the entire operation from Brighton to Woking where BT had a major printing, sorting and distribution centre. Towards the end of March we were ready to receive and process the local authorities’ data.
+
+This process involved printing each local authorities bills direct debit instructions and payment books together with the supporting documents.
+
+In the distribution centre were four high speed laser printers that worked at 180 A4 pages per minute. Payment books were guillotined stapled and assembled ready to go into the hoppers of the collating machines. These machines assembled the documents for an individual, enveloped them and sorted the envelopes into walking order for the postman. 
```

Comment:
Does it need a comma? `Payment books were guillotined, stapled ...`.

Reply:
Again yes it does